doc/interesting-queries.md

```diff
@@ -298,6 +298,82 @@ from the treasury. These can be coalesced into single query via an SQL `union` o
 ...
 ```
 
+### Get historical UTxO set for a given timestamp
+The UTxO set is dependent on time, this will return it for a given timestamp
+```
+WITH const AS (
+  SELECT TO_TIMESTAMP('2020-10-10 17:00:00', 'YYYY-MM-DD HH24:MI:SS') AS effective_time_
+)
+
+# SELECT tx_out.address        AS address
+    ,    (tx_out.value / 1e6)  AS ada
```

Comment:
If you use `1000000` instead of `1e6` I think the result will be truncated so we avoid things like `1.00000000000000000000 ada` as below.

Reply:
I tried, but the result is the same, I could leave it in Lovelace instead.

Reply:
Yes please.